src/main/java/edu/kis/powp/jobs2d/drivers/adapter/Job2dDriverToAbstractDriverAdapter.java

```diff
@@ -0,0 +1,21 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.powp.jobs2d.AbstractDriver;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class Job2dDriverToAbstractDriverAdapter extends AbstractDriver implements Job2dDriver {
+    private Job2dDriver job2dDriver;
+    private int x;
+    private int y;
```

Comment:
Abstract driver already has x, y (available using getX and getY) and setPosition implemented